lib/transport/selenium-webdriver/index.js

```diff
@@ -121,7 +121,7 @@ class SeleniumWebdriver extends WebdriverTransport {
         this.agent_ = opt_agent || null;
         //eslint-disable-next-line
         const options = url.parse(serverUrl);
-
+        options.port = (options.port && typeof options.port === 'string') ? Number(options.port) : options.port;
```

Comment:
why not just options.port = Number(options.port)?

Reply:
Just to handle the edge case where `port` is not provided in config.

Reply:
Actually, it could happen that the port is not set, so we need to set a default (80/443), if the port is undefined or null.  The `serverUrl` could be something like `https://some.selenium.host/` and then the port is dropped completely.

Reply:
Added a commit to take default port from `ServiceBuilder` when it is absent in config.